### PR TITLE
Expand prettier coverage to cypress tests

### PR DESCRIPTION
**Pull request by PaulChinkwende**
Altered the linting script so it runs on cypress test folder as well as cypress tests in the storybook directory.

* [x] Fixes https://github.com/bbc/simorgh/issues/52
* [x] Up-to-date with latest - `git pull --rebase origin latest`
* [x] Runs locally - `npm run dev` & [http://localhost:7080/](http://localhost:7080/)
* [ ] Tests added for new features
* [x] Tests pass - `npm test`
* [x] E2E tests pass - `npm run test:e2e` (requires server running see [README](https://github.com/bbc/simorgh#tests) for details)

* [ ] Test engineer approval

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -14,6 +14,10 @@
       "jsx": true
     }
   },
+  "globals":  {
```
Comment (jamesbhobbs) [inline .eslintrc]: I think it's a little unsafe to declare these as globals. For example in the /src/ directory any variable called cy or Cypress will pass linting even though it will not be defined. Can you add this to only the /cypress/ directory?

File: cypress/integration/article_spec.js [hunk 2]
```diff
@@ -1,23 +1,21 @@
 describe('News Article', () => {
```
Comment (jamesbhobbs) [inline cypress/integration/article_spec.js:3]: Please make sure to explain these changes to @jamesbrumpton

**Review by jamesbhobbs — CHANGES_REQUESTED** [review]

File: cypress/integration/article_spec.js (hunk 2 above)
Comment (PaulChinkwende) [reply cypress/integration/article_spec.js:3]: Will Do

File: .eslintrc (hunk 1 above)
Comment (PaulChinkwende) [reply .eslintrc]: Done

File: .eslintrc [hunk 3]
```diff
@@ -14,8 +14,5 @@
       "jsx": true
     }
   },
-  "plugins": ["react", "json", "jsx-a11y"],
-  "rules": {
-    "prettier/prettier": ["error", {singleQuote: true, trailingComma: "all"}]
-    }
+  "plugins": ["prettier", "json"],
```
Comment (jamesbhobbs) [inline .eslintrc:17]: why is this file changed?

**Review by jamesbhobbs — APPROVED** [review]
@PaulChinkwende this looks really good, well done. I'm merged it now because it's been open too long and is hard to keep up to date. Any errors subsequently found should be fixed-fowards.

File: .eslintrc (hunk 3 above)
Comment (PaulChinkwende) [reply .eslintrc:17]: @benjaminhobbs React and jsx-a11y plugins are brought in through the air-bnb style guide so we do not need them here. Putting prettier here lets us define its rules in another file our .prettierrc
